monitoring/mongodb/dashboard.py

```diff
@@ -0,0 +1,1353 @@
+    [
+        Target(
+            expr=f'sum by (pod) (rate(mongodb_ss_wt_cache_eviction_worker_thread_evicting_pages{{job=~"$job", namespace="{NAMESPACE}"}}[$__rate_interval]))',
+            legendFormat="{{pod}}",
+        )
+    ],
+    unit=UNITS.OPS_PER_SEC,
+    fillOpacity=0,
+    showPoints="auto",
+    legendDisplayMode="table",
+    legendPlacement="right",
+    legendValues=["mean", "max", "min"],
+)
+
+wt_cache_io = mongodb_timeseries(
+    "Cache I/O",
+    [
+        Target(
+            expr=f'sum(rate(mongodb_ss_wt_cache_bytes_{operation}_cache{{job=~"$job", namespace="{NAMESPACE}"}}[$__rate_interval]))',
+            legendFormat=f"{operation}_cache",
+        )
+        for operation in ["written_from", "read_into"]
+    ],
+    unit="binBps",
+    fillOpacity=0,
+    showPoints="auto",
+)
+
+evictions_app_threads = mongodb_timeseries(
+    "Evictions from application threads",
+    [
+        Target(
+            expr=f'rate(mongodb_ss_wt_cache_modified_pages_evicted_by_application_threads{{job=~"$job", rs_state="{RS_STATE_PRIMARY}", namespace="{NAMESPACE}"}}[$__rate_interval])',
+            legendFormat="{{pod}}",
+        )
+    ],
+    legendDisplayMode="table",
+    legendPlacement="right",
+    unit=UNITS.OPS_PER_SEC,
+    fillOpacity=0,
+    showPoints="auto",
+)
+
+wiredtiger_cache = mongodb_timeseries(
+    "Cache Size",
+    [
+        Target(
+            expr=f'sum({metric}{{job=~"$job", namespace="{NAMESPACE}"}}) by ()',
+            legendFormat=legend,
+        )
+        for metric, legend in [
+            ("mongodb_mongod_wiredtiger_cache_max_bytes", "max"),
+            ("mongodb_mongod_wiredtiger_cache_bytes", "used"),
+            ("mongodb_ss_wt_cache_tracked_dirty_bytes_in_the_cache", "dirty"),
+        ]
+    ],
+    unit=UNITS.BYTES,
+)
+
+# Create the dashboard
+dashboard = (
+    Dashboard(
+        title="MongoDB",
+        editable=True,
+        sharedCrosshair=True,
+        refresh="",
+        tags=[],
+        timezone="browser",
+        inputs=[
+            DataSourceInput(
+                name="DS_PROMETHEUS",
+                label="Prometheus",
+                pluginId="prometheus",
+                pluginName="Prometheus",
+            ),
+            ConstantInput(
+                name="namespace",
+                label="Namespace",
+                description="Namespace associated with the Zenko instance",
+                value="zenko",
+            ),
+            ConstantInput(
+                name="job",
+                label="Job",
+                description="Name of the MongoDB job, used to filter only the MongoDB instances.",
+                value="data-db-mongodb-sharded",
+            ),
+        ],
+        templating=Templating(
+            [
+                Template(
+                    dataSource=DATASOURCE,
+                    name="job",
+                    label="Job",
+                    query='label_values(mongodb_up{namespace="zenko"},job)',
```

Comment:
this should use the namespace and job inputs to filter the returned jobs:

```suggestion
                    query='label_values(mongodb_up{namespace="${namespace}", job=~"${job}-.*"},job)',
```